src/main/java/org/osc/sdk/controller/api/SdnRedirectionApi.java

```diff
@@ -226,10 +227,11 @@ void updateInspectionHook(InspectionHookElement existingInspectionHook)
      * </p>
      *
      * @param inspectionPort  provides the inspection port to be created
+     * @return an element
```

Comment:
also can we use registerNetworkElement instead of this method? maybe this can be deprecated?